cloudbridge/cloud/providers/gce/provider.py

```diff
@@ -157,6 +160,38 @@ def parse_url(self, url):
                 out.parameters[parameter] = m.group(index + 1)
             return out
 
+    def get_resource_url_with_default(self, resource, identifier):
```

Comment:
Looks like this is never called with a URL. If that's the case maybe something like `resource_name_to_url` or `get_resource_url_by_name` is better?

Reply:
As discussed in the meeting, I changed the parameter name identifier to url_or_name to make it more clear.